packages/jupyter-ai/jupyter_ai/chat_handlers/default.py

```diff
@@ -45,5 +45,8 @@ def create_llm_chain(
 
     async def process_message(self, message: HumanChatMessage):
         self.get_llm_chain()
-        response = await self.llm_chain.apredict(input=message.body, stop=["\nHuman:"])
+        with self.pending("Thinking"):
```

Comment:
should the message be "Generating response" instead of "Thinking" to better reflect what is happening?